CIP-0128/README.md

```diff
@@ -0,0 +1,143 @@
+---
+CIP: 128
+Title: Preserving Order of Transaction Inputs
+Category: Plutus
```

Comment:
```suggestion
Category: Ledger
```
@solidsnakedev as you recall from last hour's meeting that this is similar to other CIPs on the `Ledger` to `Plutus` spectrum which support Plutus implementations but are really implemented in the Ledger.  This category will be especially useful since Ledger has already given positive feedback here and (as I understand it) will be responsible for the implementation.

This goes somewhat against what's currently published in CIP-0084 regarding the [ledger-script interface](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0084#the-ledger-script-interface) so I would welcome @WhatisRT @lehins to post what they feel about both this suggestion & the quotation above (maybe the latter needs to be updated based on 2 years of Plutus, Ledger & CIP process evolution).

Reply:
This is definitely more relevant to Ledger, since almost all of the changes will be on the Ledger side and affect Plutus only slightly.